mp/src/game/shared/momentum/weapon/weapon_mom_rocketlauncher.cpp

```diff
@@ -24,8 +30,19 @@ PRECACHE_WEAPON_REGISTER(weapon_momentum_rocketlauncher);
 #ifdef GAME_DLL
 static MAKE_TOGGLE_CONVAR(mom_rj_center_fire, "0", FCVAR_ARCHIVE,
                           "If enabled, all rockets will be fired from the center of the screen. 0 = OFF, 1 = ON\n");
+static MAKE_CONVAR(
+    mom_rj_sounds, "1", FCVAR_ARCHIVE | FCVAR_CLIENTDLL,
+    "Toggles between the TF2 rocket and weapon sounds and the Momentum ones. 0 = None, 1 = Momentum, 2 = TF2\n", 0, 2);
+static MAKE_TOGGLE_CONVAR(
+    mom_rj_viewmodel, "0", FCVAR_ARCHIVE | FCVAR_CLIENTDLL,
```

Comment:
`FCVAR_CLIENTDLL` doesn't make sense to use here, this is inside of a `#ifdef GAME_DLL` (server DLL def check) block?

Reply:
Also if this is a toggle cvar between using momentum or TF2, why not name this var a little bit better:
`mom_rj_use_tf_viewmodel`